.gitignore

```diff
@@ -47,3 +47,5 @@ package-lock.json
 npm-debug.log*
 .eslintcache
 browser_logs.log
+# Local secret env vars
```

Comment:
see comment above. also, if you want to make a case for inclusion in _this_ PR, i'm down to hear it!

Reply:
👍